pyanaconda/screen_access.py

```diff
@@ -236,6 +238,19 @@ def get_screen_option_changed(self, screen_name, option_name):
                           option_name, screen_name)
             return False
 
+    def synchronize_config_file_state(self):
```

Comment:
I don't really like the name `synchronize`, because the method just updates some values in the config file.

Reply:
I see it as synchronizing the config state with state of the DBus modules - but might as well change it to update. :)